### ISSUE TITLE
I have problem with reproducing results on Table 3 in the paper

### ISSUE THREAD

**Issue by daehuikim** [clear-nus/edc#16]
Thanks for sharing your excellent work!
This framework support easy-to-use interface for OIE.
While run the provided code, I found the evaluation results reported on paper is not reproducible from given codes.
Below are my command and evaluation results.
In Table 3, Mistral-7b's EDC only results are around 70~75 F1 scores. But my reimplementation results are around 60~68 F1 score.
Is there any missing point to run this code?

If the following result occurred due to a simple mistake on my part, I apologize. The following result was reproduced using the provided example without any code modifications.

P.S. results are below

run.sh
```
OIE_LLM=mistralai/Mistral-7B-Instruct-v0.2
SD_LLM=mistralai/Mistral-7B-Instruct-v0.2
SC_LLM=mistralai/Mistral-7B-Instruct-v0.2
SC_EMBEDDER=intfloat/e5-mistral-7b-instruct
DATASET=webnlg

python ./edc/run.py \
    --oie_llm $OIE_LLM \
    --oie_few_shot_example_file_path "./edc/few_shot_examples/${DATASET}/oie_few_shot_examples.txt" \
    --sd_llm $SD_LLM \
    --sd_few_shot_example_file_path "./edc/few_shot_examples/${DATASET}/sd_few_shot_examples.txt" \
    --sc_llm $SC_LLM \
    --sc_embedder $SC_EMBEDDER \
    --input_text_file_path "./edc/datasets/${DATASET}.txt" \
    --target_schema_path "./edc/schemas/${DATASET}_schema.csv" \
    --output_dir "./edc/output/${DATASET}_target_alignment2" \
    --logging_verbose
```
evaluation result
```
-----------------------------------------------------------------
Total scores
-----------------------------------------------------------------
Ent_type
Correct: 9057 Incorrect: 395 Partial: 0 Missed: 2657
Spurious: 3081 Possible: 12109 Actual: 12533
Precision: 0.6733144758898088 Recall: 0.6971969166082691
F1: 0.6822980025502801
-----------------------------------------------------------------
Partial
Correct: 8495 Incorrect: 0 Partial: 957 Missed: 2657
Spurious: 3081 Possible: 12109 Actual: 12533
Precision: 0.6733853998657783 Recall: 0.6921377404033325
F1: 0.6802055117401999
-----------------------------------------------------------------
Strict
Correct: 8152 Incorrect: 1300 Partial: 0 Missed: 2657
Spurious: 3081 Possible: 12109 Actual: 12533
Precision: 0.6165704160097993 Recall: 0.6295141322121
F1: 0.6214987366704255
-----------------------------------------------------------------
Exact
Correct: 8495 Incorrect: 957 Partial: 0 Missed: 2657
Spurious: 3081 Possible: 12109 Actual: 12533
Precision: 0.6432979644612441 Recall: 0.6563069376313945
F1: 0.6480043743533582
-----------------------------------------------------------------
Scores per tag
-----------------------------------------------------------------
Subjects
-----------------------------------------------------------------
Ent_type
Correct: 3302 Incorrect: 160 Partial: 0 Missed: 557
Spurious: 640 Possible: 4019 Actual: 4102
Precision: 0.7524137662539905 Recall: 0.7661761270731138
F1: 0.7564665576578681
-----------------------------------------------------------------
Partial
Correct: 3235 Incorrect: 0 Partial: 227 Missed: 557
Spurious: 640 Possible: 4019 Actual: 4102
Precision: 0.768852682395079 Recall: 0.7783228217706143
F1: 0.7715350774741104
-----------------------------------------------------------------
Strict
Correct: 3087 Incorrect: 375 Partial: 0 Missed: 557
Spurious: 640 Possible: 4019 Actual: 4102
Precision: 0.717200031145371 Recall: 0.7185237094136884
F1: 0.7176010277972437
-----------------------------------------------------------------
Exact
Correct: 3235 Incorrect: 227 Partial: 0 Missed: 557
Spurious: 640 Possible: 4019 Actual: 4102
Precision: 0.7499026707155649 Recall: 0.7530950712450362
F1: 0.7507397025617067
-----------------------------------------------------------------
Predicates
-----------------------------------------------------------------
Ent_type
Correct: 2844 Incorrect: 19 Partial: 0 Missed: 1215
Spurious: 1297 Possible: 4078 Actual: 4160
Precision: 0.6339834929533598 Recall: 0.6456435412286848
F1: 0.6379506345869346
-----------------------------------------------------------------
Partial
Correct: 2676 Incorrect: 0 Partial: 187 Missed: 1215
Spurious: 1297 Possible: 4078 Actual: 4160
Precision: 0.624065638869423 Recall: 0.6325624854006073
F1: 0.6270497547302032
-----------------------------------------------------------------
Strict
Correct: 2672 Incorrect: 191 Partial: 0 Missed: 1215
Spurious: 1297 Possible: 4078 Actual: 4160
Precision: 0.6116950868177217 Recall: 0.6148096239196449
F1: 0.612940901658491
-----------------------------------------------------------------
Exact
Correct: 2676 Incorrect: 187 Partial: 0 Missed: 1215
Spurious: 1297 Possible: 4078 Actual: 4160
Precision: 0.6120844039554622 Recall: 0.6155103947675776
F1: 0.6134392275947987
-----------------------------------------------------------------
Objects
-----------------------------------------------------------------
Ent_type
Correct: 2911 Incorrect: 216 Partial: 0 Missed: 885
Spurious: 1144 Possible: 4012 Actual: 4271
Precision: 0.6537867580264216 Recall: 0.6764774585377248
F1: 0.6599704118975317
-----------------------------------------------------------------
Partial
Correct: 2584 Incorrect: 0 Partial: 543 Missed: 885
Spurious: 1144 Possible: 4012 Actual: 4271
Precision: 0.6461139661034545 Recall: 0.6642139686989021
F1: 0.6514365802382621
-----------------------------------------------------------------
Strict
Correct: 2393 Incorrect: 734 Partial: 0 Missed: 885
Spurious: 1144 Possible: 4012 Actual: 4271
Precision: 0.5565677801136806 Recall: 0.5572296192478393
F1: 0.5568169430818345
-----------------------------------------------------------------
Exact
Correct: 2584 Incorrect: 543 Partial: 0 Missed: 885
Spurious: 1144 Possible: 4012 Actual: 4271
Precision: 0.5953048353188507 Recall: 0.6017285680915674
F1: 0.5974382932336681
-----------------------------------------------------------------
Full triple scores
-----------------------------------------------------------------
Precision: 0.12343767680188177 Recall: 0.06490213948801941
F1: 0.07850344185810115
```

**Comment by bzhangj13zzz** [comment]
Thank you for the issue and sharing the results. To clarify: in our experiments, we always used gpt-3.5-turbo for the schema definition and canonicalization stage, hence the slight performance difference.

The test of different LLMs only happens at the extraction stage. I believe this is stated in our paper (4.1 EDC models). We did this because we considered OIE to be the key upstream task and we did not have enough time and resources to test out different permutations of models for different stages.

I hope this answers your questions!
